src/main/java/com/cincialert/controllers/UserController.java

```diff
@@ -1,53 +1,94 @@
 package com.cincialert.controllers;
 
+import com.cincialert.dao.UserDAO;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import com.cincialert.model.User;
+import com.cincialert.dto.User;
 import com.cincialert.service.UserService;
 
+import java.io.IOException;
 import java.util.List;
 
 @RestController
+
+
 @RequestMapping("/users")
 public class UserController {
 
     @Autowired
     private UserService userService;
 
+
+
     @PostMapping
     public ResponseEntity<User> createUser(@RequestBody User user) {
-        return ResponseEntity.ok(userService.createUser(user));
+        User newUser = userService.save(user);
```

Comment:
The response entities needed overhauled to explore more information in the API endpoints. This was just added on top of fixing your layer issues.